src/main/java/teammates/ui/newcontroller/CreateAccountAction.java

```diff
@@ -63,10 +63,11 @@ protected ActionResult execute() {
         }
 
         List<InstructorAttributes> instructorList = logic.getInstructorsForCourse(courseId);
-        String joinLink = Config.getFrontEndAppUrl(Const.ActionURIs.INSTRUCTOR_COURSE_JOIN)
+        String joinLink = Config.getFrontEndAppUrl(Const.WebPageURIs.JOIN_PAGE)
                 .withRegistrationKey(StringHelper.encrypt(instructorList.get(0).key))
                 .withInstructorInstitution(instructorInstitution)
-                .toAbsoluteString();
+                .withParam(Const.ParamsNames.ENTITY_TYPE, "instructor")
+                .toString();
```

Comment:
Should use `toAbsoluteString`? or the email will only contains relative string path.

```
</p>

<p>You can click the link below to 'join' TEAMMATES.</p>
<a href=/web/join?key=A7438D1F2F40650A6733A8C15F617D615CEB6F4F99E17630884C180F902AF0F8058B28E62DFE0586CE20C4B2EC8B1F23&instructorinstitution=asdfasdf&entitytype=instructor target="_blank" rel="noopener noreferrer">/web/join?key=A7438D1F2F40650A6733A8C15F617D615CEB6F4F99E17630884C180F902AF0F8058B28E62DFE0586CE20C4B2EC8B1F23&instructorinstitution=asdfasdf&entitytype=instructor</a>

<p>
```

Reply:
Good catch, looks like I copied-and-pasted wrongly here.